locale/po/grasswxpy_zh.po

```diff
@@ -20720,11 +20720,6 @@ msgstr "标签："
 #~ msgid "Automatically hightlight selected features in map display"
 #~ msgstr "地图显示窗口中自动高亮所选择要素"
 
-#~ msgid "Systematic contiguos"
-#~ msgstr "系统连续"
-
-#~ msgid "Systematic non contiguos"
-#~ msgstr "系统不连续"
```

Comment:
same as above, just that I'm not able to read this :)

Reply:
As I said, I searched the file and the exact translation exists for the fixed source strings.